charts/zookeeper-operator/templates/operator.yaml

```diff
@@ -39,10 +43,16 @@ spec:
               fieldPath: metadata.name
         - name: OPERATOR_NAME
           value: {{ template "zookeeper-operator.fullname" . }}
+        {{- if .Values.additionalEnv }}
+{{ toYaml .Values.additionalEnv | indent 8 }}
+        {{- end }}
         {{- if .Values.resources }}
         resources:
 {{ toYaml .Values.resources | indent 10 }}
         {{- end }}
+      {{- if .Values.additionalSidecars }}
+{{- include "chart.additionalSidecars" . | indent 6 }}
+      {{- end }}  
```

Comment:
Could we also add `.Values.additionalSidecars` and `.Values.additionalVolumes` in the same way in which `.Values.additionalEnv` has been added, instead of defining them in the helpers file ?

Reply:
@SrishT 👍 

additionalSidecars and additionalVolumes moved from helper class